shell/probe/install.ps1

```diff
@@ -10,8 +10,15 @@ param(
   [String]$endpointId=$env:computername
 )
 
+function FromEnv { param ([string]$envVar, [string]$default)
```

Comment:
nit: move params to next line just to be consistent with other function headers

Reply:
also a small nit: be consistent across the files wrt if FromEnv function comes before or after the TLS  setting